apps/dialtone-documentation/docs/.vuepress/theme/components/DialtoneLogo.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <router-link
-    class="d-btn h:d-bgc-transparent d-py6 d-w216 d-jc-flex-start"
+    class="d-btn d-btn--muted h:d-bgc-transparent d-py6 d-px12 d-w216 d-jc-flex-start"
     title="Go back to the homepage"
     to="/"
   >
-    <svg-loader :illustration="true" name="dialpad-logo" />
+    <dt-stack gap="400" direction="row">
+      <dt-icon
+        name="dialpad-ai-color-reversed"
+        size="500"
+      />
+      <span class="d-fs-300 d-fw-bold">Dialtone</span>
```

Comment:
Bold feels far too overpowering. Suggest semibold, maybe even medium.

```suggestion
      <span class="d-fs-300 d-fw-semibold">Dialtone</span>
```

Reply:
Let's leave it for now. We can always reduce it in the future. 😄